core/data/src/main/java/uk/gov/gchq/gaffer/data/element/EdgeDirection.java

```diff
@@ -16,9 +16,12 @@
 
 package uk.gov.gchq.gaffer.data.element;
 
+import uk.gov.gchq.koryphe.Summary;
+
 /**
  * Enumerated type denoting the directionality of an {@link Edge}.
  */
+@Summary("The directed of an edge")
```

Comment:
"The direction" I think makes more sense? This summary is also identical almost to the Summary in DirectedType